lib/cisco_node_utils/bgp_neighbor_af.rb

```diff
@@ -205,64 +205,54 @@ def default_as_override
 
     # -----------------------
     # <state> capability additional-paths receive <disable>
-    # Nvgens as True -OR- True with 'disable' keyword
-    def cap_add_paths_receive_get
-      val = config_get('bgp_neighbor_af', 'cap_add_paths_receive', @get_args)
-      return nil if val.nil?
-      (val.shift[/disable/]) ? 'disable' : true
-    end
-
-    def cap_add_paths_receive
-      cap_add_paths_receive_get.nil? ? false : true
-    end
-
-    def cap_add_paths_receive_disable
-      cap_add_paths_receive_get.to_s[/disable/] ? true : false
-    end
-
-    def cap_add_paths_receive_set(state, disable=false)
-      set_args_keys(state:   (state ? '' : 'no'),
-                    disable: (disable ? 'disable' : ''))
-      config_set('bgp_neighbor_af', 'cap_add_paths_receive', @set_args)
-    end
-
-    def default_cap_add_paths_receive
-      config_get_default('bgp_neighbor_af', 'cap_add_paths_receive')
+    #   :enable  = capability additional-paths receive
+    #   :disable = capability additional-paths receive disable
+    #   :inherit = no capability additional-paths receive
```

Comment:
Can you add a note to indicate where we inherit from?  This comment makes it look like we are removing the feature.